### PR TITLE
Add tutorial on fitting scintillation velocities

### PR DESCRIPTION
**Pull request by rikvl**
@mhvk
Some questions I had about the tutorial on fitting velocities:
- Should the `get_chi2()` function wrap (call) the model function `model_dveff_abs()` or not (the way it is now)?
- Right now the amplitudes `amp_p` and `amp_e` can be negative during the fitting. Should that be avoided?
  This could be done with an additional prior function, or it could be included in `fit_wrapper()`, with something like this (although I don't know what the effect of infinites would be on `scipy.optimize.minimize()`):
```
  if pars_quantity_dict['amp_p'] < 0 or pars_quantity_dict['amp_e'] < 0:
      return np.inf
```
- I had to upgrade to a new version of matplotlib to use `matplotlib.colors.CenteredNorm` (requires mpl 3.4.0).
  Not sure if a note about that should be added, and if so, where.
- Should there be some discussion for situations when velocities are close to zero, and if so, where do you think would be a good place for that?

### CODE REVIEW

**Comment by rikvl** [comment]
@mhvk -- I changed to the `amp_s * sin(ph) + amp_c * cos(ph)` formulation for the fitting.

**Review by mhvk — COMMENTED** [review]
Very nice! A few small things

File: docs/tutorials/extract_phys_pars.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+******************************
```
[the remark this answers is not in the document]
Comment (rikvl) [reply docs/tutorials/extract_phys_pars.rst:1]: Yes, it's just a placeholder here.

File: docs/tutorials/fit_velocities.rst [hunk 2]
```diff
@@ -0,0 +1,815 @@
+********************************
+Fitting scintillation velocities
+********************************
+
+This tutorial describes how to fit a phenomenological model to a time series of
+scintillation velocities for a pulsar in a circular orbit and a single
+one-dimensional screen. It requires a file containing synthetic data:
+:download:`fake-data-J0437.npz <../fake-data-J0437.npz>`
```
[the remark this answers is not in the document]
Comment (rikvl) [reply docs/tutorials/fit_velocities.rst]: Done.

File: docs/tutorials/fit_velocities.rst [hunk 3]
```diff
@@ -0,0 +1,815 @@
...
+        'loc': 'left', 
+        'x': 0.01,
+        'y': 1.0,
+        'pad': -14
+    }
+
+Set known parameters
+====================
+
+Set the pulsar's orbital period :math:`P_\mathrm{b}` and time of ascending node
```
[the remark this answers is not in the document]
Comment (rikvl) [reply docs/tutorials/fit_velocities.rst:96]: Added.

File: docs/tutorials/fit_velocities.rst [hunk 4]
```diff
@@ -0,0 +1,815 @@
+********************************
+Fitting scintillation velocities
+********************************
+
+This tutorial describes how to fit a phenomenological model to a time series of
+scintillation velocities for a pulsar in a circular orbit and a single
+one-dimensional screen. It requires a file containing synthetic data:
```
[the remark this answers is not in the document]
Comment (rikvl) [reply docs/tutorials/fit_velocities.rst]: Done.

**Comment by rikvl** [comment]
@mhvk - I implemented the comments, and immediately changed the symbols from Omega_s to xi, and from xi_p/xi_E to chi_p/chi_E.

Also had a (very quick) look around to see which convention is used for position angles when there is a 180 deg ambiguity, and found that 0 to 180 deg is generally used, not -90 to 90 deg. So we now use 0 < xi < 180 deg.

**Review by mhvk — APPROVED** [review]
OK, great. Let's get this in!

**Pull request merged**